src/components/app/details/cdDetails/deploymentHistoryDiff/DeploymentHistoryDiffView.tsx

```diff
@@ -137,8 +160,18 @@ export default function DeploymentHistoryDiffView({
                         className="code-editor-header-value pl-16 pr-16 pt-12 pb-12 fs-13 fw-6 cn-9 bcn-0 dc__top-radius-4 dc__border-bottom"
                         data-testid="configuration-link-comparison-body-heading"
                     >
-                        {baseTemplateConfiguration?.codeEditorValue?.['displayName']}
+                        <span>{baseTemplateConfiguration?.codeEditorValue?.['displayName']}</span>
+                        {isVariablesAvailable ? (
```

Comment:
we can reWrite this as `{isVariablesAvailable && (`

Reply:
resolved